app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -89,6 +88,32 @@ class Overview extends React.Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    const { finishedTransformationPlans, addNotificationAction } = this.props;
+    const freshMigrations = finishedTransformationPlans.filter(
```

Comment:
We might need to do something else to monitor for `freshMigrations`

Since the `finishedTransformationPlans` are being passed down into `Overview` as props, and they have an initial value of `[]`, a user is going to see a toast for every finished migration when first logging in (which is why we're seeing the two in `mockMode`)

Reply:
Is that not the desired behavior? Might need ux to weigh in on when they want to see them...